server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -559,8 +559,9 @@ export class AgenticChatController implements ChatHandlers {
             // Phase 3: Request Execution
             this.#validateRequest(currentRequestInput)
             const response = await session.generateAssistantResponse(currentRequestInput)
-            this.#debug(`Q Model Response: ${loggingUtils.formatObj(response, { depth: 5 })}`)
```

Comment:
can we still keep this?

Reply:
is any of the non-metadata info useful? This is what I see: 
```
{
  "$metadata": {
    "httpStatusCode": 200,
    "requestId": "046690bd-6d69-4e1f-a68a-8f302dfd0264",
    "attempts": 1,
    "totalRetryDelay": 0
  },
  "conversationId": "",
  "generateAssistantResponseResponse": {
    "options": {
      "messageStream": {
        "options": {
          "inputStream": {},
          "decoder": {
            "headerMarshaller": {},
            "messageBuffer": [],
            "isEndOfStream": false
          }
        }
      }
    }
  }
}
```

Reply:
I think you are right, they are not useful because the response is being streamed back, we can remove it here. Do we have logs in response streaming (during parsing)?

Reply:
We have some for when we identify tool use events, and when errors occurs. See implementation [here](https://github.com/aws/language-servers/blob/584450fba054f4bbcd702ab49a58e45d9abd3d1f/server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatEventParser.ts#L87-L195). 

Are there any other important cases that might be helpful for debugging?